migrator.php

```diff
@@ -1107,6 +1113,12 @@ public function orders( $args, $assoc_args ) {
 					continue;
 				}
 
+				// Mask phone number in test mode.
+				if ( $mode === 'test' ) {
```

Comment:
We should use Yoda comparison

Reply:
Thanks. I have changed this at both places where I have used this check : https://github.com/woocommerce/migrator-cli/pull/5/commits/e7b3b587c941cf839528a0b2f8f968805269d3bb

Reply:
Marking this as resolved